diff --git a/node_modules/expo-location/ios/EXLocation.m b/node_modules/expo-location/ios/EXLocation.m
index 548b20c..d8f3814 100644
--- a/node_modules/expo-location/ios/EXLocation.m
+++ b/node_modules/expo-location/ios/EXLocation.m
@@ -19,6 +19,7 @@ NS_ASSUME_NONNULL_BEGIN
         @"speed": @(location.speed),
         },
     @"timestamp": @([location.timestamp timeIntervalSince1970] * 1000),
+    @"mocked": @(location.sourceInformation.isSimulatedBySoftware)
     };
 }
 
